bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/handler/Utils.java

```diff
@@ -122,4 +122,17 @@ public static int[] convertVersionStringToIntArray(String input) {
         }
         return Arrays.stream(input.split("\\.")).mapToInt(Integer::parseInt).toArray();
     }
+
+    /**
+     * Capitalizes the first character of a string.
+     *
+     * @param input The input string.
+     * @return The string with the first character in uppercase.
+     */
+    public static String capitalizeFirstLetter(String input) {
```

Comment:
Please remove this utility method and update it's callers with
https://www.openhab.org/javadoc/latest/org/openhab/core/util/stringutils#capitalize(java.lang.String)